app/views/layouts/application.html.haml

```diff
@@ -2,7 +2,7 @@
 %html
   %head
     %meta{content: 'text/html; charset=UTF-8', http: {equiv: "Content-Type"}}/
-    %title Kibbles n' Bites - Fine Pet Cuisine
+    %title PlayDate
```

Comment:
If you get problems with displaying flashes when removing the devise views, then you need to display flashes in this file

Reply:
Understood.